politeiawww/cmd/politeiawwwcli/commands/newdcc.go

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) 2017-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package commands
+
+import (
+	"bufio"
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/decred/politeia/politeiad/api/v1/mime"
+	cms "github.com/decred/politeia/politeiawww/api/cms/v1"
+	www "github.com/decred/politeia/politeiawww/api/www/v1"
+	"github.com/decred/politeia/util"
+)
+
+// NewDCCCmd submits a new dcc.
+type NewDCCCmd struct {
+	Args struct {
+		Type        uint     `positional-arg-name:"type"`            // 1 for Issuance, 2 for Revocation
+		Attachments []string `positional-arg-name:"attachmentfiles"` // DCC attachment files
+	} `positional-args:"true" optional:"true"`
+	Type          string ``
+	NomineeUserID string `long:"nomineeuserid" optional:"true" description:"The UserID of the Nominated User"`
+	Statement     string `long:"statement" optional:"true" description:"Statement in support of the DCC"`
+	Domain        string `long:"domain" optional:"true" description:"The domain of the nominated user"`
+}
+
+// Execute executes the new dcc command.
+func (cmd *NewDCCCmd) Execute(args []string) error {
```

Comment:
I'm getting this error when trying to submit a dcc using this command:
`SignMerkleRoot: invalid digest: file: digest:`

The request data being sent looks fine.
```
{
  "type": 1,
  "nomineeuserid": "a00d06bf-d7ac-4785-bb8f-19ff57f77706",
  "statement": "i support",
  "domain": 1
}
```

Reply:
Did you rebuild piwwwcli?  I'm pretty sure I fixed the issue that was causing that previous, and was able to successfully submit a newdcc.

Reply:
oops, you're right.